src/components/Users/UserResetPassword.tsx

```diff
@@ -1,216 +1,219 @@
-              {...field("old_password")}
-              name="old_password"
-              label={t("current_password")}
-              className="col-span-6 sm:col-span-3"
-              type="password"
-              required
-              aria-label={t("current_password")}
-            />
-            <div className="col-span-6 sm:col-span-3">
-              <TextFormField
-                {...field("new_password_1")}
-                name="new_password_1"
-                label={t("new_password")}
-                type="password"
-                className="peer col-span-6 sm:col-span-3"
-                required
-                aria-label={t("new_password")}
-              />
-              <div
-                className="text-small mb-2 hidden pl-2 text-secondary-500 peer-focus-within:block"
-                aria-live="polite"
-              >
-                {validateRule(
-                  field("new_password_1").value?.length >= 8,
-                  t("password_length_validation"),
-                  !field("new_password_1").value,
-                )}
-                {validateRule(
-                  field("new_password_1").value !==
-                    field("new_password_1").value?.toUpperCase(),
-                  t("password_lowercase_validation"),
-                  !field("new_password_1").value,
-                )}
-                {validateRule(
-                  field("new_password_1").value !==
-                    field("new_password_1").value?.toLowerCase(),
-                  t("password_uppercase_validation"),
-                  !field("new_password_1").value,
+  return (
+    <div className="overflow-hidden rounded-lg bg-white px-4 py-5 shadow sm:rounded-lg sm:px-6">
+      <div className="mb-4 flex justify-start">
+        <Button
+          onClick={() => setIsEditing(!isEditing)}
+          type="button"
+          id="change-edit-password-button"
+          className="flex items-center gap-2 rounded-sm border border-gray-100 bg-white px-3 py-1.5 text-sm text-[#009D48] shadow-sm hover:bg-gray-50"
+        >
+          <CareIcon
+            icon={isEditing ? "l-times" : "l-edit"}
+            className="h-4 w-4"
+          />
+          {isEditing ? t("cancel") : t("change_password")}
+        </Button>
+      </div>
+      {isEditing && (
+        <Form {...form}>
+          <form onSubmit={form.handleSubmit(handleSubmitPassword)}>
+            <div className="space-y-4">
+              <FormField
+                control={form.control}
+                name="old_password"
+                render={({ field }) => (
+                  <FormItem>
+                    <FormLabel>{t("old_password")}</FormLabel>
+                    <FormControl>
+                      <TextFormField
+                        {...field}
+                        type="password"
+                        onChange={(value) => {
+                          field.onChange(value.value);
+                        }}
+                      />
```

Comment:
shadcn's input component instead. FormField components are not intended to be used along with shadcn's form component.